<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deviceInformationWidget</class>
 <widget class="QWidget" name="deviceInformationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>适用用户：</string>
          </property>
          <property name="buddy">
           <cstring>applyUserBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="applyUserBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>通讯链路：</string>
          </property>
          <property name="buddy">
           <cstring>linkBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="linkBox"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>型号代码：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="modelIDlineEdit"/>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>设备类型：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="primaryClass"/>
        </item>
        <item>
         <widget class="QComboBox" name="minorClass"/>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>厂商编码：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="manufactureIDLineedit"/>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>URL 地址：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="urllineEdit"/>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="updateCheckbox">
          <property name="text">
           <string>是否支持在线升级</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="synCheckBox">
          <property name="text">
           <string>支持时间同步</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="sleepCheckBox">
          <property name="text">
           <string>休眠</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="otherparaCheckBox">
          <property name="text">
           <string>第三方兼容设备地址</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPushButton" name="savepushButton">
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>设备名称：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="namelineEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="lable_1">
          <property name="text">
           <string>文件版本：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileVerEdit"/>
        </item>
       </layout>
      </item>
      <item row="10" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
